posthog/celery.py

```diff
@@ -192,6 +195,42 @@ def pg_table_cache_hit_rate():
             pass
 
 
+@app.task(ignore_result=True)
+def pg_plugin_server_query_timing():
+    from posthog.internal_metrics import gauge
+
+    with connection.cursor() as cursor:
+        try:
+            cursor.execute(
+                """
+                SELECT
+                    substring(query from 'plugin-server:(\\w+)') AS query_type,
+                    total_time as total_time,
+                    (total_time / calls) as avg_time,
+                    min_time,
+                    max_time,
+                    stddev_time,
+                    calls,
+                    rows as rows_read_or_affected
+                FROM pg_stat_statements
+                WHERE query LIKE '%%plugin-server%%'
+                ORDER BY total_time DESC
+                LIMIT 50
+                """
+            )
+
+            for row in cursor.fetchall():
+                row_dictionary = {description.name: value for description, value in zip(cursor.description, row)}
```

Comment:
This is handy, but I didn't actually know `cursor.description` so I was wondering for a moment why the elements of `cursor.description` are also named `cursor` here – `column.name: value` would be a bit clearer